cves/CVE-2018-13304.yml

```diff
@@ -246,8 +254,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: commit message lists 'avcodec'
+  name: avcodec
```

Comment:
Same nit as previous CVE. Seems right, but explain why this file is in `avcodec`, and what the `avcodec` area generally does so the reader makes a logical connection and understands the project structure better.

Ex: this file is a decoder, and is thus in the library that handles video decoding.